internal/database/migrations/82_remove_legacy_cluster_template_seed.up.sql

```
@@ -0,0 +1,14 @@
+--
+-- Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+-- the License. You may obtain a copy of the License at
+--
+--   http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+-- an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+-- specific language governing permissions and limitations under the License.
+--
+
+delete from cluster_templates where id = 'ocp_4_17_small';
```

SQLFluff (4.2.2)

[error] The 'where' keyword should always start a new line.

(LT14)